Providers/Modules/Plugins/SecurityBaseline/conf/security_baseline.conf

```diff
@@ -10,7 +10,7 @@
 <source> 
     type exec
     tag oms.security_baseline
-    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
+    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
```

Comment:
Yes, we reproduced the problem on our machines and verified the fix. I will continue with tests in INT environment.